### PR TITLE
Fix code style/grammar on synthetic event warning

### PR DESCRIPTION
**Pull request by spicyj**

### CODE REVIEW

**Comment by spicyj** [comment]
@jimfb (from #4635)

File: src/renderers/dom/client/syntheticEvents/SyntheticEvent.js [hunk 1]
```diff
@@ -111,14 +114,17 @@ assign(SyntheticEvent.prototype, {
...
-        'for more information.'
+      warning(
+        event,
+        'This synthetic event is reused for performance reasons. If you\'re ' +
+        'seeing this, you\'re calling `stopPropagation` on a ' +
+        'released/nullified synthetic event. This is a no-op. See ' +
+        'https://fb.me/react-event-pooling for more information.'
       );
     }
-    if (!event) return;
```
Comment (zpao) [inline src/renderers/dom/client/syntheticEvents/SyntheticEvent.js]: Did lint really not warn about this?

Comment (jimfb) [inline src/renderers/dom/client/syntheticEvents/SyntheticEvent.js]: Everything was green.

Comment (zpao) [inline src/renderers/dom/client/syntheticEvents/SyntheticEvent.js]: Yea I saw, was just surprised. Turns out I totally missed part of the ESLint migration to 1.0 guide (where they dropped having a default config enabled). #4746 will handle that.

**Pull request merged**